runtime/drivers/file/repo.go

```diff
@@ -97,7 +97,7 @@ func (c *connection) Put(ctx context.Context, instID, filePath string, reader io
 		return err
 	}
 
-	f, err := os.Create(filePath)
+	f, err := os.OpenFile(filePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o644)
```

Comment:
@rakeshsharma14317 : this would have a side effect on appending sources as well, In case of sources and other code artifacts, we would want to preserve existing behaviour of Put as it would break the application. 

Another way to fix this would be to add a new method Append to to RepoStore, that appends instead of replacing the file. and preserve the existing behaviour of put method.

Reply:
Yeah make sense, this would impact the other artifacts like sources etc, Added the new Append method which we can use for this purpose and changing it for `.gitignore` for now

Reply:
We should perhaps add a mode argument similar to how file system APIs work.